pkg/R/RDD.R

```diff
@@ -1377,8 +1282,243 @@ setMethod("setName",
             x
           })
 
+#' @examples
```

Comment:
top, fold, aggregate, pipe, name, setName are already present in RDD.R in the sparkr-sql branch. The only things to preserve are `zipWithUniqueId` and `zipWithIndex`